src/components/EnvelopeSkeleton.vue

```diff
@@ -57,7 +57,7 @@
 							</div>
 
 							<div class="list-item-content__inner__details">
-								<div v-if="showDetails" class="list-item-content__inner__details__details">
+								<div :class="['list-item-content__inner__details__details', { 'list-item-content__inner__details__details__hidden': showDetails }]">
```

Comment:
```suggestion
								<div :class="['list-item-content__inner__details__details', { 'list-item-content__inner__details__details--hidden': showDetails }]">
```

nit: I think *hidden* can be seen as BEM *modifier*